### ISSUE TITLE
Get an error in subscription

### ISSUE THREAD

**Issue by SilencerWeb** [prisma/prisma#2920]
**Describe the bug**
On frontend I get an error `Cannot read property 'task' of undefined` in the subscription. I didn't get it before, I guess it started to behave like that after I upgraded dependencies or added new subscriptions for another type (same error - `Cannot read property 'user' of undefined`). I tried to downgrade dependencies, revert commits both on frontend and backend - it didn't help.

Here is an example of subscription and I think the error is here because somehow `subscription` is `undefined`: 
```js
const taskCreated = {
  subscribe: (_, args, context, info) => {
    return context.prisma.subscription.task({ where: { mutation_in: ['CREATED'] } }, info);
  },
};
```

**To Reproduce**
1. `git clone https://gitlab.com/SilencerWeb/dailyq-api.git`
2. `cd dailyq-api && git checkout development && yarn install && yarn start`

To make this example work, you need to provide variables `DEVELOPMENT_PRISMA_ENDPOINT`(url to prisma endpoint), `DEVELOPMENT_PRISMA_SECRET`(any string) and `DEVELOPMENT_APP_SECRET`(any string) in `.env` and then run `prisma deploy`.

**Expected behavior**
Shouldn't throw an error `Cannot read property 'task' of undefined` and I should recieve data in the subscription on frontend

**Screenshots**
Screenshot of the error on frontend:
![image](https://user-images.githubusercontent.com/20713191/44099374-eae96496-9ffb-11e8-9cf8-f2d8ff52afc9.png)

**Versions (please complete the following information):**
 - OS: macOS High Sierra 10.13.6
 - "graphql-yoga": "^1.16.0"
 - "prisma-binding": "^2.1.4"
 - "graphql-cli": "^2.16.5"
 - "prisma": "^1.13.7"
[package.json before and after upgrade](https://gitlab.com/SilencerWeb/deadline-tracker-api/commit/415427513b38d519a7a65fe9fa2ad089dc01d7e5#7030d0b2f71b999ff89a343de08c414af32fc93a), doesn't work in both cases.

**UPDATE:**
[Here](https://gitlab.com/SilencerWeb/karma-api) is another repository with the same structure and same versions of packages. Subscriptions defined identically in both repositories, but in this one they work.

Also, when I make `console.log(context)` inside the subscription, I get such message: `(node:12649) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated` in the first repository but I do not get it in the second repository.

**Comment by SilencerWeb** [comment]
Okay, it is very very weird.

I COPIED `node_modules` from the second repository where subscriptions work to the first one and they started working (even that versions in `package.json` are same!). But after removing `node_modules` and runninng `yarn install` they stopped working again. Wtf?

[Commit with COPIED `node_modules`](https://gitlab.com/SilencerWeb/dailyq-api/commit/b55ed0ba0c4b61d9e1ef59c7f86bcba0c037656b)
[Commit after re-installing `node_modules`](https://gitlab.com/SilencerWeb/dailyq-api/commit/dca09aedc9fbd660cfef76e7f14bb914b1df4c93)

P.S. I don't think it is because of my laptop, it works like that even on heroku

**Comment by frankdugan3** [comment]
I am having the same issue. Wiping `yarn.lock` and `node_modules` made no difference for me.

I'm running `prisma@1.14.1` and the matching container on Ubuntu 18.04.

**Comment by SilencerWeb** [comment]
@frankdugan3 do you use `graphql-yoga` for the server? If yes, could you try to clone my repository, copy `node_modules` and try to test your subscriptions without installing/deleting other dependencies? Only subscriptions.

Here is a single command to clone repository and checkout to certain branch:
`git clone https://gitlab.com/SilencerWeb/dailyq-api.git && cd dailyq-api && git checkout node_modules && git checkout b55ed0ba0c4b61d9`. After this copy `node_modules` and paste them in your project.

Also, a little bit more explanation of this behavior: https://github.com/prisma/graphql-yoga/issues/432

**Comment by frankdugan3** [comment]
@SilencerWeb I'm using `apollo-server-express`, so I think our situation is a little different. Probably narrows it down to `prisma-binding` or `prisma`.

**Comment by andreasenberg** [comment]
I've been experiencing similar problems myself using prisma-binding 2.1.4 and apollo-server 2.0.4.  I happened to have another test project I've been playing around with running apollo-server 2.0.0 and prisma 2.1.3 and using those I was not getting this error.

So I tried downgrading prisma-binding and apollo-server to the same versions as in my test project and doing so I was not getting this error anymore. 
Then I again upgraded apollo-server to 2.0.4 and it seems to be working as expected. So from my end this seems to be an issue introduced in prisma-binding 2.1.4.

Would be interesting to know if you are able to reproduce this if you've downgraded to "prisma-binding": "2.1.3".

**Comment by frankdugan3** [comment]
Downgrading to `prisma-binding@2.1.3` solves the issue for me.

**Comment by SilencerWeb** [comment]
Downgrading to `prisma-binding@2.1.3` solves the issue for me too. @andreasenberg thank you so much, I had been trying to solve this issue for a week :)))

But note this: if your `prisma-binding` version in `package.json` is like that `"prisma-binding": "^2.1.3"`, issue will still exist, but if it is like that `"prisma-binding": "2.1.3"`, issue will be solved :)

**Comment by maury91** [comment]
I already found the root cause of this issue in https://github.com/prisma/prisma-binding/issues/229#issuecomment-414149550 , will be good if a contributor of this project (like @marktani  ) can point what fix implement

**Comment by marktani** [comment]
This should be fixed now, as https://github.com/prisma/prisma-binding/issues/229 is fixed.

**Comment by divyenduz** [comment]
This is released in `2.1.4` 🎉
